### DISCUSSION TITLE
[2022년 10월 28일] : 2부 이것도 모르세요? 까지 리뷰

### DISCUSSION THREAD

**Discussion by WhorideChicken** [ReadingKing/ReadingThinkingSharing#26] [Show and tell]
## 2부 이것도 모르세요? 까지 리뷰
 # DH
우원님이 말한것 처럼 이번 챕터는 소프트 스킬 위주의 인사이트를 얻을 수 있었다.
조엘 테스트에서 리더쉽에 부분은 스킬적 요소도 있겠지만 개인적으로는 경험이 정말 중요하다고 생각한다. 팀의 리더를 맡고 사람을 이끌며 많은 시행착오를 겪으며 얻는 노하우들은 직접 경험해 보지 않으면 얻을 수 없다고 생각합니다.

- HY : 여러 대표님들을 만나보면 대표들 마다 리더쉽 스타일이 다르고 배우면 안되는 사람들을 통해 역으로 배우기도 합니다.(독재자)

* DH : 각자가 생각하는 관리자의 가장 중요한 능력은 어떤 것인가?

- HY : 사람을 보는 눈을 기르는것이 가장 중요하다고 생각합니다. 
- WW : 팀원 개개인을 파악하고 시간을 투자해서 팀 분위기를 잘 이끌면 좋을거 같습니다.
- MH : 결정력이라 생각합니다.  의견이 다들 갈릴 때 확실한 결정을 해서 관리자로써 신뢰도 올릴 수 있을 것이다.

- DH : 저는 개인적으로 상황을 이해하는 능력이라고 생각합니다. 

- DH : 디자인 결과물과 개발 결과물은 맥락이 다르기 떄문에 이번 챕터에서 사례가 올바르지 않다고 생각합니다. 

코드에 대한 리뷰는 맥락없이 자유롭게 이뤄져도 생각해도 된다고 봅니다. 열린 피드백이 좋은 방향을 이끈다고 생각합니다.

* DH : 나의 결과물을 공유하여 유의미한 피드백을 얻었던 경험이 있었는지?

- HY : 창업했을 때 멘토로 계셨던 교수님과 토의를 한 결과 리소스 및 굳이 유니티가 필요없는 사이트는 Flutter를 이용해서 개발하는 방식을 배웠습니다.
- WW : 기획가 디자인 된 UI를 실제로 디바이스에 올려보고 예기치 못한 피드백을 받게 되고 다른 앱을 개발할 때도 사용할 수 있는 로직을 개발 할 수 있었습니다.
- MH : 우원님이나 하연님이 피드백 준 내용을 기억해 뒀다가 다음에 계속해서 활용하고 있습니다. 아직 본인은 스스로 부족하다 느껴 매번 공유 드리고 피드백을 받고 싶다.

* DH : 누군가를 설득하기 위해 각자 가지고 있는 팁이 있을까요?

- HY : 감정적인 호소가 가능하면 스스로를 불쌍하게 포장하며 불가능하면 대안책을 여러개 들고가서 최악의 상황을 가정하면서 이야기를 이끌어간다.
- WW : 정확한 수치나 도표,  권위자의 멘트등을 곁들여서 커피나 음식(조공)을 같이 마시며 말합니다.
- MH : 우원님 의견에서 이어지는데 그런데도 안먹히면 미연시 하는 느낌으로 말랑말랑하게 상대방을 공략한다.

* DH : 누군가를 멘토링 해본 경험이 있을까요?
* 저를 멘토링 해주신분이 우원님의 방식과 비슷한 성향이었던거 같습니다. 차이점은 계획을 본인이 짜게 뒀습니다.(기획만 한 달 하기도 했습니다.)

- WW : 누군가를 멘토링 해보지 않았지만 하게 된다면 그사람 맞춤형의 체계적으로 스케쥴을 짜서 난이도를 조절할거 같습니다.(과제나 소규모 프로젝트를 같이 해본다.)
- HY : 해당 사람의 니즈를 먼저 파악해서 진행을 했었습니다. 열정적인 학생(멘티)의 역할도 중요하기 하다고 생각합니다. 같이 성장할 기회 이기도 하고 그 과정에서 스스로도 깨달음을 얻을 수 있습니다.
- MH : 하연님과 비슷한 생각입니다. 저는 자기소개서 컨펌을 해줬었습니다. 그 친구들도 사람마다 천차만별인데 결과만 받아가는 사람이 있는가 반면 자신이 계속해서 열심히 발전 시키는 사람이 있기도 하다 전자의 경우에는 의욕이 떨어지지만 후자의 경우 신나서 멘토링을 해줄 수 있었습니다. 의지가 있는 멘티라면 방치가 되지는 않을 것이다. 
  

# MH 
 이번 파트를 읽으면서 초반부터 동의하지 못하는 부분이 있어서 집중을 못하다가 후반부에는 집중 할 수 있었습니다. 완벽한 팀에 대한 생각을 많이 하게 되었습니다.
ex) 삼전 개발팀은 완벽한 팀일까?

- WW : 남 탓 금지 나를 믿고동료를 믿자

* MH : 좋은 팀을 찾기 위해 회사를 선택할 떄 주로 보는 기준이 궁금하다.

- HY : 현재 나의 커리어나 기술과 관련된 곳을 찾습니다.
- DH : 하고싶었던  일을 찾아 첫번째 회사를 선택했습니다. 처음에는 모든걸 고려하지는 못했고(일과 경험을 쌓는게 중요했기 때문에) 두번쨰는 메타버스에 관심이 많았기 떄문에 MMO게임서버에 관심이 많았어서 메타버스는 게임서버를 할 수 있을 줄 알고 선택했다가 금방 흥미가 식어서 세번째 회사는 두번째 회사에서 실망했던 부분을 메꾸려고 찾아보려고 했습니다. 복지, 개발 환경 등등
- WW : 첫번째 회사는 막연하게 뽑히는대로 갔습니다. 그 다음은 대우(급여와 환경)을 따져서 지금 회사를 입사하게 되었습니다. 지금 당장 고르라고 하면 팀원과 회사의 분위기 그리고 내가 하고싶은 분야인지를 고려할거 같습니다.

* MH : 개인적으로 궁금한데 MBTI를 어떻게 생각하는지 그리고 각자의 MBTI가 궁금합니다.(ISTJ입니다.)

- HY : INTJ인데 재미로 보는 통계라고 생각하니까 재밌게 느껴집니다. 
본인은 MBTI를 볼 떄 T/F만 봅니다.
T가 이성과 F는 감정이기 떄문에 사람별로 다른 대응을 해드려야 한다. 
- DH : ISTJ 이며 MBTI를 아예 신뢰를 안하고 있습니다.
- WW : ENFJ 이지만 “나”는 유일하기 떄문에 결만 같은 사람들이라고 생각합니다. 스몰 토크용으로 좋은 소재라 생각합니다.

# HY
린 스타트업을 보면 탄식부터 나옵니다. 창업 했을 때 외부의 조언을 구했는데 항상 린 스타트업을 추천하는데 많이 당했었다. 속도를 빠르게 낼 수 있으면 사용해도 되지만 그렇지 않으면 좋지 않다.

* HY : 다른 분들이 추천하는 의미있는 소음 활동이 있나요?
팀에게 좋은 영향을 주는 소프트 스킬들이 있는지

- DH : 새로운 방법은 없는거 같고 스몰 토크를 좋아합니다. 현재 팀분위기가 딱딱한데 편하게 대화는 잘 못하지만 그와중에도 성향이 비슷한 사람끼리 잡담하는데 그게 리프레쉬도 되고 좋은거 같습니다.
- WW : 종종 유튜브 링크나(제가 재밋던거) 소재를 공유하면서 잠깐 숨 고르기를 하면서 쉬는데 여러분들이 잘 받아주셔서 저는 감사하게 생각합니다.
- MH : 업무 이야기로 대화를 건내면서 이야기를 확장 시키면서 다른 부서분들하고 친해집니다.

* HY : 만나본 사람 중에 이 사람은 정말 대화하기 어렵다 싶은 멘탈 모델이 있는지? 극복한 나름의 팁이 있다면 궁금합니다.

- DH : 자존감이 낮은 사람이 힘들었습니다. 그런분들에게는 그 사람이 싫어하는 행동은 하지 않습니다.
- WW : 본인의 작업이나 행동에 설명을 요구 할 때 두루뭉실하게 말하는 사람(명확한 설명을 못하는 사람)이 힘든데 그사람의 작업물이나 행동을 분석해서 궁금한것만 따로 체크해서 요점만 물어봅니다.
- MH : 농담을 진지하게 받아들이고 끝까지 물고 늘어지는 스타일(찐특)이 피곤하다 그런 사람을 보면서 저러지 않아야지 하고 생각합니다.

<img width="1432" alt="스크린샷 2022-10-28 오후 10 36 43" src="https://user-images.githubusercontent.com/97941811/198673967-f9ace368-152b-4135-b5d1-021133da0d37.png">

**Comment by Joseph-Cha** [comment]
https://github.com/ThinkAboutSoftware/AcademicConference/discussions/236

오늘 제안 했던 2023 Academic Conference 일정 및 선정된 책입니다 ㅎㅎ
각자 고민해 보고 이야기 나눠 봅시닷!
